.github/actions/install-skia-dependencies/action.yaml

```diff
@@ -12,10 +12,11 @@ runs:
           if: runner.os == 'macOS'
           run: brew install ninja
           shell: bash
-        - name: Upgrade LLVM for Skia build on Windows
-          if: runner.os == 'Windows'
-          run: choco upgrade llvm --version "19.1.7"
-          shell: bash
+        # Disabled because the windows build machine comes with llvm 20
+        # - name: Upgrade LLVM for Skia build on Windows
+        #   if: runner.os == 'Windows'
+        #   run: choco upgrade llvm --version "19.1.7"
+        #   shell: bash
```

Comment:
When testing, make sure that the test build runs with clean caches, as this installation of llvm is only used when building skia, i.e. when the rust cache is clear.